### PR TITLE
Added pudhina-fresh

### PR DESCRIPTION
**Pull request by ritijjain**

### CODE REVIEW

**Comment by JugglerX** [comment]
@ritijjain Is this a duplicate of this theme? https://jamstackthemes.dev/theme/pudhina/

**Comment by ritijjain** [comment]
@JugglerX It is built on top of this theme which is why they look similar.

**Comment by JugglerX** [comment]
Ok, maybe you could you describe what new functionality you've added so it's clear what the difference is between the themes?

**Comment by ritijjain** [comment]
@JugglerX Yes I agree, I have updated the feature list to make what's changed more clear.

**Review by JugglerX — APPROVED** [review]

File: content/theme/pudhina-fresh.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* Masonry style projects page with collapsible cards – [demo](https://ritijjain.github.io/pudhina-fresh/projects/).
+* Customizable post meta data – tags, published date, updated date, author.
+* Responsive mobile friendly design.
+* Code blocks powered by [highlight.js](https://highlightjs.org/) accompanied by matching styles for in-line code.
+* Create redirect pages using the `_layouts/redirect` template.
+* Support for Bootstrap elements and bootstrap compatible styles.
+* Support for Disqus comments and Google Analytics.
+* Redesigned table, heading, subheading, link, quote, and text styles.
+* Just as fast as the original – 90+ [Google PageSpeed Insights score](https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fritijjain.com%2F).
+* Read full [post on Pudhina Fresh](https://ritijjain.com/2020/09/12/pudhina-fresh.html).
```
Comment (Dimahmet) [inline content/theme/pudhina-fresh.md:36]: P

**Review by Dimahmet — COMMENTED** [review]
Duplicate of #

**Pull request merged**